templates/translate.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    <p><a href="{{ path('home') }}">{{ msg('pick-another') }}</a></p>
+    <h1 class="filename">File:{{ filename }}</h1>
```

Comment:
Should we have a space after `File:`?

Reply:
I'm not sure. It's not got a space on the [prototype](https://prtksxna.github.io/svgtranslate-prototype/solarsystem.html), and I guess is mirroring what things look like on [Commons](https://commons.wikimedia.org/wiki/File:Planets2013.svg). I also wasn't sure if 'File:' should be translated, but I think it makes sense not to.

Reply:
Oh, I see now. It's mimicing the URL/naming structure used in wiki projects. It's not really for humans. :)

Reply:
Yep. Agreed. I get it now.

Reply:
Agree with @mooeypoo. We don't need the "File:" there. @prtksxna any objections?

Reply:
Do we want to keep `File:` in the URL structure?

Reply:
>  we are asking the users to put in a filename from Commons without the File: namespace

Actually, it works if they put in a filename with or without, or a whole URL. So maybe that search message is a bit misleading. We chop off anything up to the end of any `File:` that's present in the string entered, and then do the search. (Oh, hmm... well, in #14 that's what might happen!)